mkt/reviewers/tests/test_views.py

```diff
@@ -90,6 +90,32 @@ def _attachments(self, num):
         return data
 
 
+class TestedonManagementMixin(object):
+
+    def _testedon_management_form(self, num=0):
+        """
+        Generate and return data for a management form for `num` tested on
+        platforms.
+        """
+        return {'testedon-TOTAL_FORMS': max(1, num),
+                'testedon-INITIAL_FORMS': 0,
+                'testedon-MAX_NUM_FORMS': 1000}
+
+    def _platforms(self, num, device_types=['Desktop', 'FirefoxOS'],
+                   devices=['PC', 'ZTE Open'], versions=['34', '1.3']):
```

Comment:
Add some unicode strings to devices, to catch some potential unicode errors. `ZTE Opén` for instance :)